docs/index.html

```diff
@@ -159,12 +159,11 @@
 
     <!--Register Service Worker-->
     <script>
-        if ('serviceWorker' in navigator) {
-            window.addEventListener('load', function() {
-                navigator.serviceWorker.register('/sw.js');
-            });
-        }
-
```

Comment:
!?

Reply:
see below

Reply:
If this is the only difference between the two HTML files, perhaps we could do this:

``` js
if ('serviceWorker' in navigator && window.location.hash !== "dev") {
```

and then open `index.html#dev` during development?

Reply:
I've implemented that now. It seems that `!== "#dev"` is required (I'm not into javascript).

Should the docs/eframe_template* in the repo?
(it will be created by ./build_web.sh anyway, unless you renamed the project then it's another name)

Reply:
> Very nice! But please revert the .js and .wasm artifacts - I only update those when I update egui to keep the number of large binary files in the git history to a minimum

It's reverted